### ISSUE TITLE
Quest links no longer work in Wrath 3.4.1

### ISSUE THREAD

**Issue by plusmouse** [Stanzilla/WoWUIBugs#380]
Previously using a quest link in a GameTooltip would display the quest objectives and description in the tooltip

```
/run GameTooltip:SetOwner(WorldFrame, "ANCHOR_CENTER");GameTooltip:SetHyperlink("quest:8327");GameTooltip:Show()
```
(quest used is https://www.wowhead.com/wotlk/quest=8327/report-to-lanthan-perilon)
Error is
```
1x GameTooltip:SetHyperlink(): Unknown link type
[string "=[C]"]: ?
[string "=[C]"]: in function `SetHyperlink'
[string "GameTooltip:SetOwner(WorldFrame, "ANCHOR_CENTER");GameTooltip:SetHyperlink("quest:8327");GameTooltip:Show()"]:1: in main chunk
[string "=[C]"]: in function `RunScript'
[string "@FrameXML/ChatFrame.lua"]:2201: in function `?'
[string "@FrameXML/ChatFrame.lua"]:4938: in function <FrameXML/ChatFrame.lua:4884>
[string "=[C]"]: in function `ChatEdit_ParseText'
[string "@FrameXML/ChatFrame.lua"]:4601: in function <FrameXML/ChatFrame.lua:4600>
[string "=[C]"]: in function `ChatEdit_SendText'
[string "@FrameXML/ChatFrame.lua"]:4637: in function `ChatEdit_OnEnterPressed'
[string "*ChatFrame.xml:121_OnEnterPressed"]:1: in function <[string "*ChatFrame.xml:121_OnEnterPressed"]:1>

Locals:
(*temporary) = GameTooltip {
 0 = <userdata>
 GetBackdropColor = <function> defined @SharedXML/SharedTooltipTemplates.lua:191
 SetHyperlink = <function> defined =[C]:-1
 SetGuildBankItem = <function> defined =[C]:-1
 SetBackdropBorderColor = <function> defined @SharedXML/SharedTooltipTemplates.lua:195
 SetCraftItem = <function> defined =[C]:-1
 comparing = false
 TopOverlay = Texture {
 }
 GetBackdropBorderColor = <function> defined @SharedXML/SharedTooltipTemplates.lua:199
 SetItemByID = <function> defined =[C]:-1
 SetLootItem = <function> defined =[C]:-1
 SetQuestItem = <function> defined =[C]:-1
 SetBorderBlendMode = <function> defined @SharedXML/SharedTooltipTemplates.lua:203
 SetInboxItem = <function> defined =[C]:-1
 layoutType = "TooltipDefaultLayout"
 shoppingTooltips = <table> {
 }
 SetMerchantItem = <function> defined =[C]:-1
 SetQuestLogItem = <function> defined =[C]:-1
 SetTradeTargetItem = <function> defined =[C]:-1
 SetTradePlayerItem = <function> defined =[C]:-1
 SetSendMailItem = <function> defined =[C]:-1
 updateTooltip = 0.183000
 SetInventoryItem = <function> defined =[C]:-1
 NineSlice = Frame {
 }
 TooltipBackdropOnLoad = <function> defined @SharedXML/SharedTooltipTemplates.lua:172
 SetBuybackItem = <function> defined =[C]:-1
 SetTradeSkillItem = <function> defined =[C]:-1
 SetBagItem = <function> defined =[C]:-1
 needsReset = true
 BottomOverlay = Texture {
 }
 SetBackdropColor = <function> defined @SharedXML/SharedTooltipTemplates.lua:187
 SetLootRollItem = <function> defined =[C]:-1
 SetAuctionItem = <function> defined =[C]:-1
}
(*temporary) = "quest:8327"
``

**Comment by Yttrium-tYcLief** [comment]
[According to today's hotfix](https://us.forums.blizzard.com/en/wow/t/wrath-of-the-lich-king-classic-hotfixes-updated-march-10/1500326), it seems like this might now be fixed?

**Comment by Meorawr** [comment]
It was indeed fixed in 3.4.1.48340 released err... a week ago.
